lib/OpenQA/WebAPI/Plugin/ObsRsync/Task.pm

```diff
@@ -35,7 +35,7 @@ sub run {
     my $retry_interval = $helper->retry_interval;
     my $queue_limit    = $helper->queue_limit;
     my $minion         = $app->minion;
-    my $lock_timeout   = 36000;
+    my $lock_timeout   = 360000;
```

Comment:
A 100h lock timeout? Does this make sense?

Reply:
Apparently current 10h is not enough for some cases when notification comes from OBS and the project must be synced once status is 'published', but it remains 'blocked' for many hours. So current approach for such cases is that gru job will be hanging around until project's status is 'published'. It can be improved, and should, but I don't have ready solution in mind.

Reply:
Ah, I see. Well, 100h is understandable and expectable in these situations.